packages/medusa/src/api/routes/store/variants/get-variant.ts

```diff
@@ -28,6 +28,9 @@ import { validator } from "../../../../utils/validator"
  *       externalDocs:
  *         url: https://en.wikipedia.org/wiki/ISO_4217#Active_codes
  *         description: See a list of codes.
+ * x-codegen:
+ *   method: retrieve
+ *   queryParams: StoreGetVariantsVariantParams
```

Comment:
Breaking change. client.productVariants.retrieve instead of client.products.variants.retrieve
https://github.com/medusajs/medusa/blob/5523222c5f3b7c2caa45a18c929d08d4a155a050/packages/medusa-js/src/resources/product-variants.ts#L17-L20